runtime.md

```diff
@@ -34,21 +34,19 @@ See [Query State](#query-state) for information on retrieving the state of a con
 ## Lifecycle
 The lifecycle describes the timeline of events that happen from when a container is created to when it ceases to exist.
 
-1. OCI compliant runtime is invoked with a reference to the location of the bundle.
-   How this reference is passed to the runtime is an implementation detail.
-2. The container's runtime environment MUST be created according to the configuration in [`config.json`](config.md).
-   Any updates to `config.json` after container is running MUST not affect the container.
-3. The prestart hooks MUST be invoked by the runtime.
-   If any prestart hook fails, then the container MUST be stopped and the lifecycle continues at step 8.
-4. The user specified process MUST be executed in the container.
-5. The poststart hooks MUST be invoked by the runtime.
-   If any poststart hook fails, then the container MUST be stopped and the lifecycle continues at step 8.
-6. Additional actions such as pausing the container, resuming the container or signaling the container MAY be performed using the runtime interface.
+1. OCI compliant runtime's `create` command is invoked with a reference to the location of the bundle and a unique identifier.
+   How these references are passed to the runtime is an implementation detail.
+2. The container's runtime environment (namespaces, mounts, etc.) MUST be created according to the configuration in [`config.json`](config.md).
```

Comment:
On Thu, Apr 14, 2016 at 01:46:32PM -0700, Vish Kannan wrote:
> > +2. The container's runtime environment (namespaces, mounts, etc.)
> > MUST be created according to the configuration in
> > [`config.json`](config.md).
> 
> nit: It's worth detailing what the `sandbox` or `environment`
> **MUST** contain.

My preference here is “everything that's not in ‘process’, while also
moving rlimits back out of ‘process’ [1].

[1]: https://github.com/opencontainers/runtime-spec/commit/84bb61924ba1f34902927d98a1774435929cd051#commitcomment-16959154